src/plugins/vis_type_timeseries/public/application/components/vis_types/timeseries/vis.js

```diff
@@ -161,6 +161,10 @@ export class TimeseriesVisualization extends Component {
     const yAxis = [];
     let mainDomainAdded = false;
 
+    const allSeriesHavePercentFormatter = seriesModel.every(
+      (seriesGroup) => seriesGroup.formatter === 'percent'
```

Comment:
I'm just wondering if your code will work correctly when using `bytes` instead of `percent` . Looks like your code should be more generic

Reply:
Thanks for the note, I've updated changes to take into account other types of formatting